src/components/books/Books.js

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
+import Book from '../book/Book';
+import AddNewBook from '../form/AddNewBook';
 
 function Books() {
-  const [books] = [
+  const books = [
     {
       title: 'The Hunger Games',
-      Authur: 'Suzanne Collins',
-      Category: 'Action',
+      authur: 'Suzanne Collins',
+      category: 'Action',
+      completed: false,
       id: 1,
     },
     {
       title: 'Dune',
-      Authur: 'Frank Herbert',
-      Category: 'Science Fiction',
+      authur: 'Frank Herbert',
+      category: 'Science Fiction',
+      completed: false,
       id: 2,
     },
     {
       title: 'Capital in the Twenty First Century',
-      Authur: 'Thomas Piketty',
-      Category: 'Economy',
+      authur: 'Thomas Piketty',
+      category: 'Economy',
+      completed: false,
       id: 3,
     },
   ];
```

Comment:
- [ ] Let's remove the hardcoded books. I think we should be mapping through the books on the Redux Store instead.